<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>musicplayerpage</class>
 <widget class="QMainWindow" name="musicplayerpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>musicplayerpage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_files">
        <property name="text">
         <string>Files</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_devices">
        <property name="text">
         <string>Devices</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_internet">
        <property name="text">
         <string>internet</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ignore_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_back">
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_home">
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_forward">
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="playlist"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_creatPlaylist">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_savePlaylist">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_loadPlaylist">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_played">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="time">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_remaning">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_mute">
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="volum">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="volumlabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_shufle">
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_prev">
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_play">
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_next">
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_mode3">
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="currentSongLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMusic">
    <property name="title">
     <string>Music</string>
    </property>
    <addaction name="actionopen_file"/>
    <addaction name="separator"/>
    <addaction name="actionprevios_song"/>
    <addaction name="actionplay"/>
    <addaction name="actionnext_trak"/>
    <addaction name="separator"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <widget class="QMenu" name="menuShuffel_mode">
     <property name="title">
      <string>Shuffel mode</string>
     </property>
     <addaction name="actionDon_t_shuffel"/>
     <addaction name="actionshuffel_all"/>
    </widget>
    <widget class="QMenu" name="menuRepeat_mode">
     <property name="title">
      <string>Repeat mode</string>
     </property>
     <addaction name="actionDon_t_repeat"/>
     <addaction name="actionrepeat_trak"/>
    </widget>
    <addaction name="actionRemove_Track"/>
    <addaction name="separator"/>
    <addaction name="menuShuffel_mode"/>
    <addaction name="menuRepeat_mode"/>
    <addaction name="separator"/>
    <addaction name="actionadd_playlist"/>
    <addaction name="actionsave_playlist"/>
    <addaction name="actionload_playlist"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuMusic"/>
   <addaction name="menuPlaylist"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen_file">
   <property name="text">
    <string>open file</string>
   </property>
  </action>
  <action name="actionprevios_song">
   <property name="text">
    <string>previous trak</string>
   </property>
  </action>
  <action name="actionplay">
   <property name="text">
    <string>play</string>
   </property>
  </action>
  <action name="actionnext_trak">
   <property name="text">
    <string>next trak</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionAdd_Track">
   <property name="text">
    <string>Add Track</string>
   </property>
  </action>
  <action name="actionRemove_Track">
   <property name="text">
    <string>Remove Track</string>
   </property>
  </action>
  <action name="actionDon_t_shuffel">
   <property name="text">
    <string>Don't shuffel</string>
   </property>
  </action>
  <action name="actionshuffel_all">
   <property name="text">
    <string>shuffel all</string>
   </property>
  </action>
  <action name="actionDon_t_repeat">
   <property name="text">
    <string>Don't repeat</string>
   </property>
  </action>
  <action name="actionrepeat_trak">
   <property name="text">
    <string>repeat track</string>
   </property>
  </action>
  <action name="actionadd_playlist">
   <property name="text">
    <string>add playlist</string>
   </property>
  </action>
  <action name="actionsave_playlist">
   <property name="text">
    <string>save playlist</string>
   </property>
  </action>
  <action name="actionload_playlist">
   <property name="text">
    <string>load playlist</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
